server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetric.js

```diff
@@ -23,6 +23,9 @@ import classNames from 'classnames';
 import Modal from '../../../../components/controls/Modal';
 import Select from '../../../../components/controls/Select';
 import Tooltip from '../../../../components/controls/Tooltip';
+import DropdownIcon from '../../../../components/icons-components/DropdownIcon';
+import BubblePopup, { BubblePopupPosition } from '../../../../components/common/BubblePopup';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'BubblePopupPosition'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3149&issues=AWIgFTRboomSmF0fSGEl&open=AWIgFTRboomSmF0fSGEl)

[ISSUE_KEY]: AWIgFTRboomSmF0fSGEl